.travis.yml

```diff
@@ -0,0 +1,15 @@
+---
+language: python
+python:
+  - "3.8"
+
+services:
+  - postgresql
+
```

Comment:
I am not sure if we need to add 
`env:`
 ` - DATABASE_URI=postgres://postgres:@localhost:5432/test` here. @billykern @biegel @VGmt

Reply:
hmm, i don't think it's necessary since it matches the default used in the tests, but i can add so that the CI config is in one place 👍

Reply:
@ZiyiHuang96 updated ✅